packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md

```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
 1. Click **Add Initial Variable** and add the following case-sensitive variables:
 
-    | Variable Name | Example Values              | Notes                                                                                                                                                                                     |
-    |:-------------------------------|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-    | SCIMBaseURL                    | `https://example.com/scim/v2`  `https://example.com/scim/v1` | For example, if your SCIM integration is hosted on <https://example.com> and uses a prefix of /scim/v2 then the *SCIMBaseURL* value for your integration would be: `https://example.com/scim/v2`. |
-    | auth                           | Bearer abcxyz1234567890     | Basic or OAuth authorization token                                                                                                                                                          |
+    | Variable Name | Example Values | Notes |
+    |:-|:-|:-|
+    | SCIMBaseURL | `https://example.com/scim/v2`  `https://example.com/scim/v1` | For example, if your SCIM integration is hosted on <https://example.com> and uses a prefix of /scim/v2 then the *SCIMBaseURL* value for your integration would be: `https://example.com/scim/v2`. |
+    | auth | Bearer abcxyz1234567890 | OAuth 2.0 Bearer token or Basic authentication code. |
     Click **Save**.
 
-1. Select the **Initial Script** tab. Copy the contents of this [Initial Script Spec](/standards/SCIM/SCIMFiles/Initial_Script_Spec.txt) text file to your clipboard and then paste it into the script console on this tab. A `Script ON` label appears on your Test Settings page to indicate that the script is accepted and will be run before the first request in the test.
+1. In a new browser window, open the [Initial Script Spec](/standards/SCIM/SCIMFiles/Initial_Script_Spec.txt) text file and copy all the text to your clipboard.
+1. Back in the Runscope **Environment** panel, select the **Initial Script** tab, and then paste the text into the script console on this tab.
+  ![Initial Script tab](/img/oin/scim-test_init-script.png "Showing the location of the Initial Script tab")
+  
+    This script sets up a number of randomly generated "dummy" variables that are used during the testing process:
+
+    | Variable Name | Example Value | Notes |
+    |:-|:-|:-|
+    | InvalidUserEmail | `abcdefgh@atko.com` | A specific email address considered invalid by the test. |
+    | randomEmail | `Runscope300Hfluaklab151@atko.com` | A valid email address. |
+    | randomFamilyName | `Hfluaklab151` | A valid last name. |
+    | randomGivenName | `Runscope300` | A valid first name. |
+    | randomUsername | `Runscope300Hfluaklab151@atko.com` | An valid user name. |
```

Comment:
I guess it should be "A valid user name."

Reply:
Good catch - thanks!